src/components/molecules/ProfilePictureInput/ProfilePictureInput.tsx

```diff
@@ -156,6 +186,7 @@ export const ProfilePictureInput: React.FunctionComponent<ProfilePictureInputPro
         Feeling camera shy? Choose from one of these Summit profile avatars.
       </small>
       <div className="default-avatars-container">
+        {gitHubImage}
         {isLoading ? <div>Loading...</div> : avatarImages}
```

Comment:
I would probably just display these together personally. Showing one but not the others seems like a weird UX flow.

Reply:
Fixes in https://github.com/sparkletown/sparkle/pull/1667/commits/8c56ac87d7d1fd51d6b29db5df212b7a3a948efc